tests/libnet/cloudinit.go

```diff
@@ -153,8 +153,8 @@ type CloudInitInterface struct {
 }
 
 type CloudInitNameservers struct {
-	Search    []string `json:"search,omitempty,flow"`
-	Addresses []string `json:"addresses,omitempty,flow"`
+	Search    []string `json:"search,omitempty"`
```

Comment:
What linter error does this cover?

Reply:
The PR description has the full list.

It is this one: `tests/libnet/cloudinit.go:156:21: SA5008: unknown JSON option "flow" (staticcheck)`